<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlcWatcherClass</class>
 <widget class="QMainWindow" name="PlcWatcherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>82</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlcWatcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="btn1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="btn2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="btn3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="btn4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="btn5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="btn6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="btn7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="btn8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menustart">
    <property name="title">
     <string>start</string>
    </property>
   </widget>
   <widget class="QMenu" name="menustop">
    <property name="title">
     <string>stop</string>
    </property>
   </widget>
   <addaction name="menustart"/>
   <addaction name="menustop"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PlcWatcher.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>startPlc()</slot>
  <slot>stopPlc()</slot>
 </slots>
</ui>
